### PR TITLE
Reg 64/load course info

### PR DESCRIPTION
**Pull request by arianaolson419**
https://olinjs.atlassian.net/browse/REG-64
This loads all courses in the course database to the state, which is not efficient, but it minimizes backend calls. Future work might include finding a more efficient way to do this.

### CODE REVIEW

**Review by jag-123 — APPROVED** [review]
Everything looks good!

File: src/actions/actions.jsx [hunk 1]
```diff
@@ -31,11 +31,12 @@ export const updateRegisterPassword = password => ({
 // Login backend interaction
 export const receiveUser = json => ({
   type: 'RECEIVE_USER',
-  username: json.username,
-  id: json._id,
-  major: json.major,
-  plannedCourses: json.plannedCourses,
-  completedCourses: json.completedCourses,
+  username: json.user.username,
+  id: json.user.id,
```
Comment (yunhsincynthiachen) [inline src/actions/actions.jsx:35]: I don't remember what was said about this, but would you want this to be '_id'?

**Review by yunhsincynthiachen — APPROVED** [review]
Other than my one comment, everything looks good to me.

File: src/actions/actions.jsx (hunk 1 above)
Comment (yunhsincynthiachen) [reply src/actions/actions.jsx:35]: @arianaolson419 once you give me the go ahead, i'll merge this pull request

Comment (arianaolson419) [reply src/actions/actions.jsx:35]: This is the object key now. I switched it around as an experiment with the linter. You can see that it matches up with the keys of the json object being sent from the /login and /register routes. The state still receives the mongoose id for Marvin. I can switch it back if you think that's best though.

Comment (yunhsincynthiachen) [reply src/actions/actions.jsx:35]: Nope! I think it's good as is! Am merging now

**Pull request merged**